src/journey-maps/angular/services/map/map-routes.service.ts

```diff
@@ -50,14 +51,16 @@ export class SbbMapRoutesService {
         }
       }
     });
-    this._mapRouteService.updateRoute(map, mapSelectionEventService, {
-      type: 'FeatureCollection',
-      // With ES2019 we can replace this with routes.flatMap(({features}) => features)
```

Comment:
fixed in a separate PR: https://github.com/sbb-design-systems/sbb-angular/pull/1788/files#diff-c95089699f3e93d1afd29196dd53830d3d40028f7798a82e20ab37bc2cf09d9eR57